packages/solana/src/connectors/walletConnectConnector.ts

```diff
@@ -144,16 +146,56 @@ export class WalletConnectConnector extends BaseConnector implements Connector {
     return { signatures: [{ signature: base58.encode(transaction.serialize()) }] }
   }
 
-  public async sendTransaction(transactionParam: Transaction | VersionedTransaction) {
+  private async _sendTransaction(transactionParam: Transaction | VersionedTransaction) {
     const encodedTransaction = (await this.signTransaction(transactionParam)) as {
       signatures: {
         signature: string
       }[]
     }
+    // Const signedTransaction = transactionParam.serialize()
```

Comment:
What is this comment for? Is it something we'd like developers to see? If yes, could we add JSDoc?